htdocs/comm/propal/list.php

```diff
@@ -548,22 +548,22 @@
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."c_availability as ava on (ava.rowid = p.fk_availability)";
 // We'll need this table joined to the select in order to filter by sale
 if ($search_sale > 0 || (!$user->rights->societe->client->voir && !$socid)) {
-	$sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
+	$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe_commerciaux as sc ON s.rowid = sc.fk_soc";
```

Comment:
Left join has not the same result than exact join (inner join)

Reply:
Yes, if you compare comma-join with left join without considering the sorting arguments.

 But in this case, the statement was:

```
... , llx_societe_commerciaux as sc
...
WHERE...
AND s.rowid = sc.fk_soc
```

Which has *exactly* the same result as, and is more understandeable than : 

```
LEFT JOIN llx_societe_commerciaux as sc ON s.rowid = sc.fk_soc
```

Repeating myself : Please note that **comma-join is NOT equivalent to inner-join"". "The precedence of the comma operator is less than that of INNER JOIN, CROSS JOIN, LEFT JOIN, and so on" [source on MySQL manual](https://dev.mysql.com/doc/refman/8.0/en/join.html)